core/src/models/Property.ts

```diff
@@ -88,6 +88,38 @@ const STATE_TRANSITIONS = [
   },
 ];
 
+/**
+ * Metadata and methods to return the options a Property for this connection/app.
+ * Options is a method which will poll the source for available options to select (ie: names of tables or columns)
+ */
+export interface PluginConnectionPropertyOption {
+  key: string;
+  displayName?: string;
+  required: boolean;
+  description: string;
+  type: "list" | "text" | "typeahead" | "textarea" | "password" | "pending"; // TODO: WHAT ARE THE OTHER ONES;
+  primary?: boolean;
+  options: (argument: {
+    connection: any;
+    app: App;
+    appId: string;
+    appOptions: SimpleAppOptions;
+    source: Source;
+    sourceId: string;
+    sourceOptions: SimpleSourceOptions;
+    sourceMapping: SourceMapping;
+    property: Property;
+    propertyId: string;
+  }) => Promise<
+    Array<{
+      key: string;
+      description?: string;
+      default?: boolean;
+      examples?: Array<any>;
+    }>
+  >;
+}
+
```

Comment:
This shouldn't be here... it's now in `core/src/class/plugin` 

https://github.com/grouparoo/grouparoo/blob/main/core/src/classes/plugin.ts#L487-L517

Reply:
Ooh!  Yep.  Is just removing these lines and committing a way to resolve that without tangling things up more between branches?